src/main/java/com/minecolonies/coremod/client/gui/WindowBlueprintSelector.java

```diff
@@ -0,0 +1,1116 @@
+package com.minecolonies.coremod.client.gui;
```

Comment:
is this just a copy? =O

Reply:
modified copy ye, since the original is final and you didnt want to have modifications I recall :D

Reply:
Would have moved some of the bigger functions into statics or so. Or created a new abstract version on the struct side, where one is strictly "Loading" and the other part is "placement"